### ISSUE TITLE
Components: headers not passed from CloudFront correctly

### ISSUE THREAD

**Issue by josephluck** [danielcondemarin/serverless-next.js#153]
I've been converting my project to use the new component and I noticed that headers aren't forwarded from CloudFront correctly (it seems).

Steps to replicate:

Create `pages/index.js` with the following:

```javascript
const Page = () => <div>Hello, world.</div>

Page.getInitialProps = ctx)=> {
  console.log({ headers: ctx.req.headers });
  console.log({ getHeaders: ctx.req.getHeaders() });
  return Promise.resolve({});
};
```

Deploy with the component:

```bash
serverless
```

The result for both console logs in CloudWatch shows all the keys of the headers with `undefined` values:

```
{
   headers:{
      'if-none-match':undefined,
      'x-forwarded-for':undefined,
      'user-agent':undefined,
      via:undefined,
      'accept-encoding':undefined,
      pragma:undefined,
      'upgrade-insecure-requests':undefined,
      'sec-fetch-mode':undefined,
      'sec-fetch-user':undefined,
      'sec-fetch-site':undefined,
      host:undefined,
      'cache-control':undefined
   }
}
```

**Comment by danielcondemarin** [comment]
Thanks for reporting @josephluck 🙏 I've now published @1.2.2 with a fix for this 🚀

**Comment by josephluck** [comment]
Just tested it out and it's working perfectly now. Thanks for fixing so quickly! 🎉
